content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
+============
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
+for some businesses it's crucial to have an accurate inventory count.
+
+The goal of cycle counts is to count more often at key locations to keep your critical stock levels
+accurate.
+
+Configuration
+=============
+
+In Odoo, cycle counts are location-based. That means that the frequency of the counts is defined on
+the storage location. That also means that to use cycle counts you need to activate the setting
+**Storage Locations**.
+
+.. image:: cycle_counts/storage-locations-setting.png
+   :align: center
+   :alt: Enable Storage Locations in Odoo settings.
+
+Then, go to the locations via :menuselection:`Inventory --> Configuration --> Locations`.
+
+Open a location and change the Inventory Frequency. In this example we want to
```

Comment:
- Replace the sentence `Open a...Inventory Frequency.` with `Then, click on a location to open the location settings. Next, click on :guilabel:'Edit'. In the :guilabel:'Inventory Frequency (Days)' field, set the number of days.`
- Replace the sentence on Lines 25-26 `In this example...every 30 days.` with `For example, a location that needs an inventory count every 30 days would set the :guilabel:'Inventory Frequency (Days)' value to '30'.`